apps/prairielearn/src/lib/question-render.sql

```diff
@@ -169,7 +169,6 @@ SELECT
   to_jsonb(lgj) AS grading_job,
   to_jsonb(s) AS submission,
   to_jsonb(v) AS variant,
-  to_jsonb(iq) || to_jsonb(iqnag) AS instance_question,
```

Comment:
You can probably remove question from the query as well, it's no longer used in the TS (I can't comment directly, but it's in this same query). Also, the JOIN on instance_questions_next_allowed_grade() is also not necessary anymore.

Reply:
Good catch, removed in f019fd4.